src/Components/AnsibleTabs/AnsibleTabs.scss

```diff
@@ -0,0 +1,102 @@
+@import '~@redhat-cloud-services/frontend-components-utilities/files/Utilities/_all.scss';
+
+section {
+    outline: none;
+}
+
+pre {
```

Comment:
oh hey, if we're using this here as well, lets pull it outta helpers.scss and here, throw it in app.scss or something 🤔